ns/rdf-xsd.ttl

```diff
@@ -0,0 +1,299 @@
+
+xsd:integer a rdfs:Datatype ;
+  rdfs:subClassOf xsd:decimal ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#integer> ;
+  rdfs:label "integer" ;
+  rdfs:comment "Signed integers of arbitrary size. Derived from xsd:decimal with fractionDigits=0. Lexical form: optional leading sign followed by one or more decimal digits; no decimal point or exponent (e.g., '-1', '0', '+100000')." .
+
+xsd:double a rdfs:Datatype ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#double> ;
+  rdfs:label "double" ;
+  rdfs:comment "IEEE 754 64-bit floating-point. Lexical forms use decimal or scientific notation with optional leading sign and exponent; special tokens 'INF', '+INF', '-INF', and 'NaN' are permitted; '+0' and '-0' are distinct lexical forms. Mapping from lexical forms to values is specific to this datatype and its precision. Lexical form (regex-style): (\\+|-)?([0-9]+(\\.[0-9]*)?|\\.[0-9]+)([Ee](\\+|-)?[0-9]+)? |(\\+|-)?INF|NaN." .
+
+xsd:float a rdfs:Datatype ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#float> ;
+  rdfs:label "float" ;
+  rdfs:comment "IEEE 754 32-bit floating-point. Lexical forms use decimal or scientific notation with optional leading sign and exponent; special tokens 'INF', '+INF', '-INF', and 'NaN' are permitted; '+0' and '-0' are distinct lexical forms. Mapping from lexical forms to values is specific to this datatype and its precision. Lexical form (regex-style): (\\+|-)?([0-9]+(\\.[0-9]*)?|\\.[0-9]+)([Ee](\\+|-)?[0-9]+)? |(\\+|-)?INF|NaN." .
+
+xsd:date a rdfs:Datatype ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#date> ;
+  rdfs:label "date" ;
+  rdfs:comment "Calendar dates as top-open day intervals. Lexical form: YYYY-MM-DD with optional timezone. Timezone lexical forms: 'Z' or an offset of the form ±hh:mm with 00 ≤ mm ≤ 59 and |offset| ≤ 14:00 (e.g., '2025-09-27Z', '2025-09-27+02:00')." .
+
+xsd:time a rdfs:Datatype ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#time> ;
+  rdfs:label "time" ;
+  rdfs:comment "Times that recur each day or occur on some day. Lexical form: hh:mm:ss(.s+)? with optional timezone. Timezone lexical forms: 'Z' or ±hh:mm with |offset| ≤ 14:00 (e.g., '23:59:59Z', '08:15:30.5-05:00')." .
+
+xsd:dateTime a rdfs:Datatype ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#dateTime> ;
+  rdfs:label "dateTime" ;
+  rdfs:comment "Instants of time, optionally with timezone. Lexical form: YYYY-MM-DDThh:mm:ss(.s+)? with optional timezone. Values with different explicit offsets can be equal as instants. Timezone lexical forms: 'Z' or ±hh:mm with |offset| ≤ 14:00 (e.g., '2025-09-27T14:03:00+02:00')." .
+
+xsd:dateTimeStamp a rdfs:Datatype ;
+  rdfs:subClassOf xsd:dateTime ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp> ;
+  rdfs:label "dateTimeStamp" ;
+  rdfs:comment "dateTime with a required explicit timezone offset. Lexical form: YYYY-MM-DDThh:mm:ss(.s+)?(Z|±hh:mm); the timezone offset is mandatory." .
```

Comment:
Is there a reason behind the omission of examples here and following (and earlier, if I failed to notice some were missing)?